lib/routes/users.js

```diff
@@ -111,7 +111,8 @@ const routes = application => {
         if (rows.length < 1) {
           res.status(404);
```

Comment:
Instead of setting status here, can we build an error object and have our error handler deal with this? This will help ensure we don't accidentally set our status multiple times.